### PR TITLE
chore: upgrades `text-split-with-logo-grid` 

### PR DESCRIPTION
**Pull request by kendallstrautman**
🔎 [Preview](https://vault-git-ksupdates-text-split-logo-hashicorp.vercel.app/) 

### Affected Pages 
- [/home](https://vault-git-ksupdates-text-split-logo-hashicorp.vercel.app/)
- [/use-cases/identity-based-access](https://vault-git-ksupdates-text-split-logo-hashicorp.vercel.app/use-cases/identity-based-access)
- [/use-cases-/secrets-management](https://vault-git-ksupdates-text-split-logo-hashicorp.vercel.app/use-cases-/secrets-management)

---

This PR updates `react-text-split-with-logo-grid` to the latest version which removes bulk from previously importing logos from `mktg-assets`. See the package [release notes](https://github.com/hashicorp/react-components/releases/tag/%40hashicorp%2Freact-text-split-with-logo-grid%405.0.0) for full context. 

_Note_, we plan to source these logo assets from the `mktg-assets` package but this is an interim solution to upgrade the component and remove bulk.

### CODE REVIEW

**Comment by kendallstrautman** [comment]
Again, used NPM v6 here, so you'll see the lock changes. Lmk if that's an issue.

File: website/content/api-docs/system/leases.mdx
Comment (kendallstrautman) [inline website/content/api-docs/system/leases.mdx:259]: This was failing the build as it read `<required>` as needing a closing tag.

**Comment by kendallstrautman** [comment]
@jescalan feel free to merge this once approved! I don't think I have rights to merge PRs on this repo.

**Review by jescalan — APPROVED** [review]
glorious!

**Pull request merged**
